### PR TITLE
feature(gce-image): added GCE image artifact test

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Review by juliayakovlev — COMMENTED** [review]
few small questions

File: test-cases/artifacts/gce-image.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cluster_backend: 'gce'
+use_preinstalled_scylla: true
+backtrace_decoding: false
```
[the remark this answers is not in the document]
Comment (bentsi) [reply test-cases/artifacts/gce-image.yaml:3]: this is an artifact test... it runs without monitoring node

File: sdcm/sct_config.py [hunk 2]
```diff
@@ -539,14 +539,17 @@ class SCTConfiguration(dict):
              help=""),
 
         dict(name="gce_image", env="SCT_GCE_IMAGE", type=str,
-             help=""),
+             help="GCE image to use for all node types: db, loader and monitor"),
```
[the remark this answers is not in the document]
Comment (bentsi) [reply sdcm/sct_config.py:542]: in past, we had one image ID for all types of instances, now we support setting different Image IDs for different instance types: DB nodes can use one image, loaders and monitors another

[the remark this answers is not in the document]
Comment (juliayakovlev) [reply sdcm/sct_config.py:542]: Yes, but now we has 3 more parameters: SCT_GCE_IMAGE_DB , SCT_GCE_IMAGE_MONITOR and SCT_GCE_IMAGE_LOADER.
So where we use this one (SCT_GCE_IMAGE)?

[the remark this answers is not in the document]
Comment (bentsi) [reply sdcm/sct_config.py:542]: it is for backword compatibility

File: sdcm/tester.py [hunk 3]
```diff
@@ -574,7 +574,7 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
                                            **common_params)
 
         loader_additional_disks = {'pd-ssd': self.params.get('gce_pd_ssd_disk_size_loader', default=0)}
-        self.loaders = LoaderSetGCE(gce_image=self.params.get('gce_image'),
+        self.loaders = LoaderSetGCE(gce_image=self.params.get('gce_image_loader'),
```
Comment (dkropachev) [inline sdcm/tester.py:577]: Should it be gce_image=self.params.get('gce_image_loader', self.params.get('gce_image')) ?

Comment (dkropachev) [reply sdcm/tester.py:577]: Also there is another place in get_cluster_k8s_gce_minikube, where we initialize LoaderSetGCE, shouldn't we have it fixed it too ?

**Review by dkropachev — APPROVED** [review]
LGTM, after addressing comments

**Pull request merged**